### PR TITLE
Use Github workflows and add test coverage with codecov.io

### PR DESCRIPTION
**Pull request by mmorel-35**
Use Github workflows instead of travis

### CODE REVIEW

File: .github/workflows/node.js.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+      uses: actions/cache@v2
+      with:
+        path: ~/.npm
+        key: ${{ runner.OS }}-node-${{ hashFiles('**/package-lock.json') }}
+        restore-keys: |
+          ${{ runner.OS }}-node-
+          ${{ runner.OS }}-
+    - run: npm install
+    - run: npm run build
+    - run: npm run coverage
```
Comment (karellm) [inline .github/workflows/node.js.yml:29]: Shouldn't it `- run: npm test` too?

**Review by karellm — COMMENTED** [review]
I added a small comment. Also I believe you need to rebase the PR.

File: .github/workflows/node.js.yml (hunk 1 above)
Comment (mmorel-35) [reply .github/workflows/node.js.yml:29]: It does, https://github.com/i18next/i18next-parser/pull/317/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R12 
Coverage includes test

**Comment by mmorel-35** [comment]
You shall now be able to add the appropriate badges to node js workflow and one for codecov.io which both require ownership to access.

**Comment by karellm** [comment]
Do you have a pointer to the docs?

**Comment by mmorel-35** [comment]
* https://docs.github.com/en/actions/managing-workflow-runs/adding-a-workflow-status-badge
* https://docs.codecov.io/docs/status-badges

**Pull request merged**